### DISCUSSION TITLE
Old quality layout not working and hdr auto brightness not getting patched and other problems

### DISCUSSION THREAD

**Discussion by Divivats** [ReVanced/revanced-discussions#312] [Question]
I am using version 17.26.35 with old quality layout patch but the patch isn't working in the app. 
Also the hdr auto brightness patch is automatically excluded while patching.
Bug: if you use swipe control patch and exclude the full screen panel patch then the comments won't scroll in full screen.
![IMG_20220712_053601](https://user-images.githubusercontent.com/108374606/178379040-571a4edc-4a5b-4ecd-acb3-3f3f510dc031.jpg)
![IMG_20220712_053537](https://user-images.githubusercontent.com/108374606/178379048-4a0a3c41-faec-41e0-85ee-25e667770e22.jpg)

**Comment by oSumAtrIX** [comment]
> Also the hdr auto brightness patch is automatically excluded while patching.

This is intended. 

> old quality layout patch but the patch isn't working in the app

I can not reproduce, please include steps to reproduce and your environment.

**Reply by Divivats** [reply]
The patches i installed are in the above images.
, Open any video in revanced and go to quality.
Old layout will not work

**Reply by oSumAtrIX** [reply]
For me it works, log out and check if that fixes it.

**Reply by Divivats** [reply]
> For me it works, log out and check if that fixes it.

Also comment section isn't scrolling in full screen due to swipe controls.

https://user-images.githubusercontent.com/108374606/178384324-ca275305-c974-4804-97df-5f1aad728df2.mp4

**Reply by Divivats** [reply]
> For me it works, log out and check if that fixes it.

I tried logging in and out,
Clearing the data of the app,
Uninstalling and reinstalling it again,
Nothing worked

**Reply by TheJeterLP** [reply]
I can confirm this, same happened for me. 17.26.34 broke it for me too.

**Reply by TheJeterLP** [reply]
![Screenshot_2022-07-12-05-52-58-98_64ef5fc2000c1caa954c114bb372e1d5.jpg](https://user-images.githubusercontent.com/3790121/178405192-7f5d5dba-782c-4165-9a2a-ac7a70727b6c.jpg)

**Reply by bogadana** [reply]
quality works fine for me on 17.26.35

**Reply by TheJeterLP** [reply]
That is very weird then

**Reply by ronin08** [reply]
using 17.26.35, old quality patch doesn't work for me too. but there's catch, it only happens when I am logged in. after entering incognito, it works fine.

WEIRD 😬

**Reply by TheJeterLP** [reply]
> using 17.26.35, old quality patch doesn't work for me too. but there's catch, it only happens when I am logged in. after entering incognito, it works fine.
> 
> WEIRD 😬

Doesn't work in incognito mode for me

**Reply by Aunali321** [reply]
it worked fine for me when i built it but sometime later patch broke. very strange.

**Reply by ronin08** [reply]
> > using 17.26.35, old quality patch doesn't work for me too. but there's catch, it only happens when I am logged in. after entering incognito, it works fine.
> > WEIRD 😬
> 
> Doesn't work in incognito mode for me

Now not working in incognito too 🤦

**Reply by ghost** [reply]
Just some Google shit. 🤣

**Reply by ghost** [reply]
Fix (under forms of PoC) to restore Old Quality Layout is available here: https://github.com/revanced/revanced-patches/issues/178
